eventnotifier/datadog/datadog.go

```diff
@@ -52,27 +53,42 @@ func (n *Notifier) GetNotifierName() string {
 	return string(types.NotifierDriverDatadog)
 }
 
-func (n *Notifier) buildDatadogEventTags(dis v1beta1.Disruption) {
+func (n *Notifier) buildDatadogEventTags(dis v1beta1.Disruption, event corev1.Event) []string {
```

Comment:
we were previously adding as a default tags all the tags to all the disruptions. Modifying the logic to not add anything to the default tags, instead build a list of tags to use in the event.